static/app/views/partials/header.html

```diff
@@ -20,8 +23,8 @@
                     <a class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true"
                     aria-expanded="false" id="org-dropdown">{{ (activeOrg | orgNameFilter) || "Organizations" }}&nbsp;<span class="caret"></span></a>
                     <ul class="dropdown-menu">
-                        <li ng-repeat="org in orgs" ng-click="showOrg(org)">
-                            <a class="org-name">{{org | orgNameFilter}}</a>
+                        <li ng-repeat="org in $parent.orgs">
```

Comment:
Should avoid using "$parent.".
These articles talk about ways to avoid having that problem.
http://blog.thoughtram.io/angularjs/2015/01/02/exploring-angular-1.3-bindToController.html
or 
http://toddmotto.com/opinionated-angular-js-styleguide-for-teams/
A little different from each other but accomplish the same thing in the end.